lib/planner/write-plans-to-disk.js

```diff
@@ -5,13 +5,14 @@ const path = require('path');
 const configHandler = require('../util/config/config-handler.js');
 
 module.exports = function writePlansToDisk(plans) {
-  let date = Date.now();
-  let fileCount = 1;
   const outputPath = configHandler.get('outputPath');
   for (let plan of plans) {
     let filePath = path.resolve(outputPath,
-        `${date}-simulato-${fileCount++}_${plans.length}.json`);
-    // fs.writeFileSync(filePath, JSON.stringify(plan.testCase));
+        (function generateTestName(plan) {
```

Comment:
Does this need to be an IIFE? https://developer.mozilla.org/en-US/docs/Glossary/IIFE